pkg/build/api/v1beta3/types.go

```diff
@@ -385,6 +390,9 @@ type BuildRequest struct {
 
 	// Revision is the information from the source for a specific repo snapshot.
 	Revision *SourceRevision `json:"revision,omitempty"`
+
+	// Image is the image that triggered this build.
+	Image string `json:"image,omitempty"`
```

Comment:
what does this correlate to? is it always set? Is this the name of an Image API object? A DockerImageReference? The term `image` is so overloaded right now, an example or more specific name would be helpful